package.json

```diff
@@ -48,5 +48,8 @@
     },
     "volta": {
         "node": "20.9.0"
+    },
```

Comment:
This needs to be move to the devDependencies section.  With Svelte, everything is a devDependency and it is compiled in.